src/team.actions.js

```diff
@@ -3,6 +3,14 @@ const { getTeamId } = require('./team-data');
 const { makeGetRequest } = require('./jira-client');
 const { getCurrentContext } = require('./config');
 
+function getPredictability(sprints) {
+  const mean = (velocities) => velocities.reduce((sum, velocity) => sum + velocity) / velocities.length;
+  const standardDeviation = (velocities) => Math.sqrt(mean(velocities.map((velocity) => Math.pow(velocity - mean(velocities), 2))));
+
+  const velocities = sprints.map(sprint => sprint.velocity).filter(velocity => velocity > 0);
+  return (100 - (standardDeviation(velocities) / mean(velocities) * 100)).toFixed(2);
```

Comment:
no safety checks on any of these values before maths?

Reply:
This is what I get when I hurry... good catch :)